examples/multiple_buckets/variables.tf

```
@@ -22,19 +22,17 @@ variable "project_id" {
 variable "names" {
   description = "Names of the buckets to create."
   type        = list(string)
-}
-
-variable "prefix" {
-  description = "Prefix used to generate bueckt names."
-  type        = string
+  default     = ["one", "see"]
```

```suggestion
  default     = ["one", "two"]
```
int test failing to to mismatch, between folders defined in https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/7c32ee228895d1bf42a54387776c3e1fb48eae7c/test/fixtures/multiple_buckets/main.tf#L33 and default here.

---

Thanks for the catch, I was changing things around for testing.